docs/openj9_support.md

```diff
@@ -42,14 +42,13 @@ columns will be removed over time.
-| 0.36.1         | Feb 2023  | :fontawesome-solid-xmark:{: .no aria-hidden="true"}<span class="sr-only">no</span> |  :fontawesome-solid-check:{: .yes aria-hidden="true"}<span class="sr-only">yes</span> | :fontawesome-solid-xmark:{: .no aria-hidden="true"}<span class="sr-only">no</span>     |     |            |
-| 0.37.0         | Mar 2023 (2)  | :fontawesome-solid-xmark:{: .no aria-hidden="true"}<span class="sr-only">no</span> | :fontawesome-solid-xmark:{: .no aria-hidden="true"}<span class="sr-only">no</span> | :fontawesome-solid-xmark:{: .no aria-hidden="true"}<span class="sr-only">no</span>     | :fontawesome-solid-check:{: .yes aria-hidden="true"}<span class="sr-only">yes</span>     |            |
-| 0.38.0         | Apr 2023   | :fontawesome-solid-check:{: .yes aria-hidden="true"}<span class="sr-only">yes</span> | :fontawesome-solid-check:{: .yes aria-hidden="true"}<span class="sr-only">yes</span> | :fontawesome-solid-check:{: .yes aria-hidden="true"}<span class="sr-only">yes</span>     | :fontawesome-solid-xmark:{: .no aria-hidden="true"}<span class="sr-only">no</span>     |            |
-| 0.39.0         | Jun 2023 (2)  | :fontawesome-solid-xmark:{: .no aria-hidden="true"}<span class="sr-only">no</span>  | :fontawesome-solid-xmark:{: .no aria-hidden="true"}<span class="sr-only">no</span>  | :fontawesome-solid-xmark:{: .no aria-hidden="true"}<span class="sr-only">no</span>      | :fontawesome-solid-xmark:{: .no aria-hidden="true"}<span class="sr-only">no</span>      |  :fontawesome-solid-check:{: .yes aria-hidden="true"}<span class="sr-only">yes</span>          |
-| 0.40.0         | Jul 2023 (1)   | :fontawesome-solid-check:{: .yes aria-hidden="true"}<span class="sr-only">yes</span> | :fontawesome-solid-check:{: .yes aria-hidden="true"}<span class="sr-only">yes</span> | :fontawesome-solid-check:{: .yes aria-hidden="true"}<span class="sr-only">yes</span>     | :fontawesome-solid-xmark:{: .no aria-hidden="true"}<span class="sr-only">no</span>     |  :fontawesome-solid-check:{: .yes aria-hidden="true"}<span class="sr-only">yes</span>          |
+| OpenJ9 release | Release date  | JDK8 (LTS) | JDK11 (LTS) | JDK17 (LTS) | JDK19 | JDK20 |  JDK21 (LTS)  |
+|----------------|---------------|------------|-------------|-------------|-------|-------|---------|
+| 0.37.0         | Mar 2023 (2)  | :fontawesome-solid-xmark:{: .no aria-hidden="true"}<span class="sr-only">no</span> | :fontawesome-solid-xmark:{: .no aria-hidden="true"}<span class="sr-only">no</span> | :fontawesome-solid-xmark:{: .no aria-hidden="true"}<span class="sr-only">no</span>     | :fontawesome-solid-check:{: .yes aria-hidden="true"}<span class="sr-only">yes</span>     |            |          |
```

Comment:
Can we remove 0.37 from the table and remove JDK19?

Reply:
Ok. Any other changes to be done in this Supported environments topic for Release 0.40.0. Thanks!

https://eclipse-openj9.github.io/openj9-docs-staging/1126/openj9_support/

Reply:
0.40.0 looks good to me.

Reply:
So no changes in the the minimum operating system levels or operating system and compiler levels for the build systems?